app/js/ProjectPersistenceManager.js

```diff
@@ -20,10 +20,28 @@ const async = require('async')
 const logger = require('logger-sharelatex')
 const oneDay = 24 * 60 * 60 * 1000
 const Settings = require('settings-sharelatex')
+const diskusage = require('diskusage')
 
 module.exports = ProjectPersistenceManager = {
   EXPIRY_TIMEOUT: Settings.project_cache_length_ms || oneDay * 2.5,
 
+  refreshExpiryTimeout(callback) {
+    diskusage.check('/', function(err, stats) {
+      if (err) {
+        return callback(err)
```

Comment:
This will result in calling `ProjectPersistenceManager.clearExpiredProjects(err)` from the interval in app.js.  `clearExpiredProjects` expects a callback. Running `errorOfDiskUsage(err)` (line 70, 86) and `errorOfDiskUsage()` (line 90) will throw and crash clsi 10min after booting.

Let's detach the two processes and call logger.error with the error of `diskusage.check`/`refreshExpiryTimeout`.